src/release.js

```diff
@@ -316,16 +463,60 @@ module.exports = function taoExtensionReleaseFactory(baseBranch, branchPrefix, o
             await config.write(data);
         },
 
+        /**
+         * Select releasing branch
+         * - picking version-to-release CLI option and find branch with version on it
+         * - or find the biggest version and find branch with version on it
+         */
+        async selectReleasingBranch() {
+            // Filter all branches to the ones that have release in the name
+            await gitClient.fetch({'--prune': true});
```

Comment:
what's the purpose of the pruning here ?

Reply:
maybe not a good idea to clean up things under the hood (let my garbage, I'll take care)

Reply:
@ricproenca-tao do you remember if `--prune` was necessary or just a personal choice?

Reply:
Removed `--prune`